src/payload/blocks/CTA/CTA_1/config.ts

```
@@ -19,10 +19,9 @@
 				"Join the developers and founders building faster with UIFoundry. Choose your path and start building today.",
 		}),
 		callToActionPair({
-			admin: {
-				initCollapsed: true,
-			},
+			minRows: 0,
+			maxRows: 2,
```

ESLint

[error] Expected "maxRows" to come before "minRows".

(perfectionist/sort-objects)